test/cmd/delete.sh

```diff
@@ -37,17 +37,17 @@ run_kubectl_delete_allnamespaces_tests() {
   kubectl delete configmap --dry-run=client -l deletetest=true --all-namespaces
   kubectl delete configmap --dry-run=server -l deletetest=true --all-namespaces
   kubectl config set-context "${CONTEXT}" --namespace="${ns_one}"
-  kube::test::get_object_assert configmap "{{range.items}}{{${id_field:?}}}:{{end}}" 'one:'
+  kube::test::get_object_assert configmap "{{range.items}}{{${id_field:?}}}:{{end}}" 'kube-root-ca.crt:one:'
```

Comment:
scope the assertion to configmaps labeled with deletetest instead of explicitly adding references to `kube-root-ca.crt`

```
kube::test::get_object_assert 'configmap -l deletetest' "{{range.items}}{{${id_field:?}}}:{{end}}" 'one:'
```

Reply:
done